libs/experimental/langchain_experimental/language_detector/lang_detect.py

```diff
@@ -0,0 +1,34 @@
+from typing import List, Tuple
+
+from langchain_experimental.language_detector.base import LanguageDetectorBase
+
+try:
+    import langdetect
+except ImportError:
+    raise ImportError(
+        "Could not import langdetect, please install with " "`pip install langdetect`."
+    )
+
+
+class LangDetector(LanguageDetectorBase):
```

Comment:
I saw in notebooks that everyone would need to reimplement the same snippet:
```
def lang_transform_func(inputs: dict) -> dict:
    text = inputs["text"]
    language = lang_detector.detect_single_language(text)
    return {"text": text, "language": language}

lang_transform_chain = TransformChain(input_variables=["text"], output_variables=["text", "language"], transform=lang_transform_func)
```
At least it should be already defined here tbh to provide a value for me, as dev. I don't need/want to reimplement this snippet as it should work out of box smoothly for other elements.
@maks-operlejn-ds I don't remember correctly, but was there a review in your MR about not providing such functionalities out of box?